src/posts/lucky_0.30.cr

```diff
@@ -0,0 +1,187 @@
+    includes some big steps forward. Better error messages, some spring
+    cleaning, a few added methods, and the new inclusion of automated security
+    testing that we're super excited about!
+
+    We will cover a few notable differences, but you can view all the changes
+    from the [CHANGELOG](https://github.com/luckyframework/lucky/blob/main/CHANGELOG.md)
+
+    ### How to upgrade
+
+    See the [UPGRADE_NOTES](https://github.com/luckyframework/lucky/blob/main/UPGRADE_NOTES.md#upgrading-from-029-to-030).
+
+    You can also view the changes using [LuckyDiff](https://luckydiff.com/?from=0.29.0&to=0.30.0).
+    This tool helps see what has changed between versions and will be included in upgrade notes from now on.
+
+    ## Here's what's new
+
+    ### Bright Security (formerly Neuralegion)
+
+    During this release, we've worked directly with the [Bright Security](https://brightsec.com/)
+    team to bring out-of-the-box security testing support for Lucky apps!
+
+    When we say "security testing", what we mean is catching vulnerabilities such as:
+
+    * insecure cookies
+    * cross-site scripting
+    * SQL injection
+    * header tampering
+    * and more...
+
+    Bright has an API that allows you to scan your application for different vulnerabilities
+    using their [SecTester](https://github.com/NeuraLegion/sec_tester) shard. By signing up
+    for an API key, you can test specific pages of your application with CI which will help
+    prevent pushing broken code to production.
+
+    We have created a thin wrapper ([LuckySecTester](https://github.com/luckyframework/lucky_sec_tester))
+    which makes writing some of these tests with your Lucky application a little easier. For now,
+    this feature is opt-in when generating a new Lucky app, but it can be added to any existing app
+    fairly easily.
+
+    To generate a new app with the SecTester, just run `lucky init.custom your_app_name --with-sec-test`.
+    This will generate a new application with the LuckySecTester setup and ready to go, along with several
+    specs already written against the existing code. This will help to serve as a template for what you
+    can add. This will also generate a Github CI already setup to run with Bright!
+
+    Due to the time it takes to run these specs, we've placed the security specs in their own spec
+    file, and behind a compile-time flag `crystal spec -Dwith_sec_tests`. This gives you the option to
+    run them only when you really need them. (e.g. during CI).
+
+    Following the Lucky mantra of catching bugs in development, it is our hope that this new integration
+    gives you peace of mind that your production app is a lot safer and solid.
+
+    ### Faster specs
+
+    After quite a bit of refactoring, we now have our database related specs running in a transaction
+    as opposed to truncating between specs. This has shown to give at least 4x speed to your specs.
+
+    ```crystal
+    # notes on how to use this
```

Comment:
@matthewmcgarvey not sure if I'm missing something, but it doesn't look like LuckyCLI generates an app with the transactional spec helper stuff. Is it an opt-in feature?